specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -259,45 +259,53 @@ op checkKeyValues is appConfigOperation<
   OkResponse & AppConfigResponseHeaders
 >;
 
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Pre-existing API contract"
 @summary("Gets a single key-value.")
 @doc("Gets a single key-value.")
-op getKeyValue is standardOps.ResourceRead<
-  KeyValue,
-  QueryParametersTrait<{
+@get
+op getKeyValue is Foundations.ResourceOperation<
```

Comment:
Why cant we use the standard resource read operation anymore?

Reply:
Standard template will cause compiler error https://github.com/Azure/azure-rest-api-specs/pull/29887#discussion_r1685972559 so we have to change to another writing way